docs/landing-page/master.adoc

```diff
@@ -69,7 +69,7 @@ include::topics/templates/document-attributes.adoc[]
     <img src="/docs/images/launch_logo.svg" alt="Fabric8 Launcher Logo" class="pull-right">
     <h2>Fabric8 Launcher documentation</h2>
     <p class="lead">
-      Read how to get started and setup your environment and all the good stuff that you can do with Fabric8 Launcher.
+      Read how to set up your environment, use the Fabric8 Launcher, and work with cloud-native runtimes on OpenShift.
```

Comment:
We should avoid phrasal verbs, so I suggest replacing _set up_ with _configure._

Reply:
I don't know, you're the wordsmiths here :-)

Reply:
@tradej From a  stylistic POV, I agree, phrasal verbs should be used with caution. But, AaMoF, I think _set up_ is fairly acceptable here, too. My experience is that we typically refer to _configuring_ applications, for instance. But as @Ladicek says, this wording implies a broader set of actions. That being said, I'm all for stylistic consistency, and since neither of the options here are wrong, I'd go with the more stylistically acceptable one, which would be _configure_.